cmake/FindCVC4.cmake

```diff
@@ -1,4 +1,12 @@
 find_path(CVC4_INCLUDE_DIR cvc4/cvc4.h)
-find_library(CVC4_LIBRARY NAMES cvc4 )
+find_library(CVC4_LIBRARY NAMES cvc4)
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(CVC4 DEFAULT_MSG CVC4_LIBRARY CVC4_INCLUDE_DIR)
+if(CVC4_FOUND)
+    find_library(CVN_LIBRARY NAMES cvn)
+    if(CVN_LIBRARY)
+        set(CVC4_LIBRARIES ${CVC4_LIBRARY} ${CVN_LIBRARY})
```

Comment:
What is CVN?

Reply:
It's CLN - I must have garbled the name in the latest rebase, not sure how that happened.